packages/components/core/option/option.ts

```diff
@@ -183,11 +183,17 @@ export class KbqOption extends KbqOptionBase implements AfterViewChecked, OnDest
      * The displayed value of the option. It is necessary to show the selected option in the
      * select's trigger.
      */
+    @Input()
     get viewValue(): string {
```

Comment:
Стоит ли давать возможность протягивать темплейт? Часто используется. Например как в Tooltip content
```suggestion
    get viewValue(): string | TemplateRef {
```

Reply:
не, сам шаблон тега можно поменять в матчере/триггере, а тут именно про viewValue, которое берется из шаблона и его невозможно было изменить